README.md

```diff
@@ -1,3 +1,3 @@
 <summary dir=rtl align=center><summary dir=ltr /><h1>leftPadDeez</h1></summary>
-<img align=center src=https://user-images.githubusercontent.com/31113245/235832123-4a107f5b-b66a-49a8-a679-e4334e38b512.png />
+<img align=center src=./nuts.webp />
```

Comment:
<img width="204" alt="Screenshot 2023-05-05 at 9 04 59 PM" src="https://user-images.githubusercontent.com/31113245/236589994-701150fb-65eb-43fd-bb45-7654149a7c9f.png">
preview in the diff viewer doesn't work since image doesn't exist yet, but it should look exactly the same just weigh 19kb instead of 202kb (see [https://github.com/PickleNik/leftPadDeez](https://github.com/PickleNik/leftPadDeez) )